core/commands/keybase/keybase.go

```diff
@@ -0,0 +1,37 @@
+package keybase
+
+import (
+	peer "github.com/libp2p/go-libp2p-core/peer"
+	mbase "github.com/multiformats/go-multibase"
+)
+
+const KeyFormatOptionName = "ipns-base"
```

Comment:
I wouldn't make a package level `KeyFormatOptionName` that is "ipns-base". Either this package is generically dealing with keys or it's only dealing with IPNS keys.

We can keep the package as something like keybase, but then this option should be more like `IPNSKeyFormatOptionName`